server.js

```diff
@@ -14,7 +14,7 @@ const issueRoutes = require('./routes/issueRoutes.js');
 const fileRoutes = require('./routes/fileRoutes.js');
 
 // import environmental variables from our variables.env file
-require('dotenv').config({ path: 'variables.env' });
+require('dotenv').config({ path: 'config/variables.env' });
```

Comment:
Make this file name even more explicit by saying something like `variablesDevelopment`